.travis.yml

```diff
@@ -43,8 +50,28 @@ matrix:
     - rvm: 1.8.7
       gemfile: gemfiles/mime_types_2.3.gemfile
     - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.4.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.5.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.6.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.6_columnar.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.99.gemfile
+    - rvm: 1.8.7
       gemfile: gemfiles/mime_types_2.latest.gemfile
     - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_3.0.gemfile
+    - rvm: 1.9.2
+      gemfile: gemfiles/mime_types_3.0.gemfile
+    - rvm: 1.9.3
+      gemfile: gemfiles/mime_types_3.0.gemfile
```

Comment:
Looks like jruby also needs to be excluded from the >= 3.0

>  Gem::InstallError: mime-types-data requires Ruby version >= 2.0.
An error occurred while installing mime-types-data (3.2015.1120), and Bundler
cannot continue.
Make sure that `gem install mime-types-data -v '3.2015.1120'` succeeds before
bundling.